source/merge.ts

```diff
@@ -0,0 +1,93 @@
+import {URL} from 'url';
+import is from '@sindresorhus/is';
+// XO doesn't recognize `export type` for some reason.
+// eslint-disable-next-line import/named
```

Comment:
XO fails at linting for exported `type`s from `known-hook-events.ts`.

Reply:
I wasn’t sure we wanted to do this across the project, but it makes it less confusing for future contributors.